### ISSUE TITLE
Can I contribute euclidean loss function, as one of the in-built keras loss functions?

### ISSUE THREAD

**Issue by shivam05011996** [keras-team/keras#10397]
Hi,
In the deep world we live today, distances are most common metrics. Everytime, developer has to write his own custom loss function, I would like to contribute a simple euclidean distance loss function from which others can save time, one or or the other. Will it be good to contribute this cause?

**Comment by briannemsick** [comment]
See mse

**Comment by shivam05011996** [comment]
@briannemsick You mean, Mean Squared Error? I think both are different.
Also the loss values vary by far extent,  for example look at the below loss, and mse.
```
Epoch 1064/2500
999997/999997 [==============================] - 3s 3us/step - loss: 0.0118 - mean_squared_error: 3.7238e-05
Epoch 1065/2500
999997/999997 [==============================] - 3s 3us/step - loss: 0.0113 - mean_squared_error: 2.9174e-05
Epoch 1066/2500
999997/999997 [==============================] - 3s 3us/step - loss: 0.0115 - mean_squared_error: 3.0376e-05
Epoch 1067/2500
999997/999997 [==============================] - 3s 3us/step - loss: 0.0113 - mean_squared_error: 3.0870e-05
Epoch 1068/2500
999997/999997 [==============================] - 3s 3us/step - loss: 0.0111 - mean_squared_error: 2.8265e-05
Epoch 1069/2500
999997/999997 [==============================] - 3s 3us/step - loss: 0.0108 - mean_squared_error: 2.6913e-05
Epoch 1070/2500
999997/999997 [==============================] - 3s 3us/step - loss: 0.0122 - mean_squared_error: 3.8306e-05
Epoch 1071/2500
999997/999997 [==============================] - 3s 3us/step - loss: 0.0102 - mean_squared_error: 2.3993e-05

```

The mse confuses the user that, loss is very very low, but in the actual implementation of euclidean distance loss, the true value is reported.
